aurora-mysql-plugin/src/test/java/io/cdap/plugin/aurora/mysql/AuroraMysqlSinkTestRun.java

```diff
@@ -85,6 +87,8 @@ public void testDBSink() throws Exception {
       Assert.assertEquals("user2", Bytes.toString(resultSet.getBytes("TINYBLOB_COL"), 0, 5));
       Assert.assertEquals("user2", Bytes.toString(resultSet.getBytes("MEDIUMBLOB_COL"), 0, 5));
       Assert.assertEquals("user2", Bytes.toString(resultSet.getBytes("LONGBLOB_COL"), 0, 5));
+      Assert.assertEquals(3.458, resultSet.getBigDecimal("NUMERIC_COL").doubleValue(), 0.00001);
```

Comment:
seems like you should be able to compare BigDecimal objects directly instead of double values.

Reply:
similar comment for rest of the tests.

Reply:
Changed to compare `BigDecimal` objects directly.